assets/helpers/abTests/annualAmountsTest.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+
+import { type AmountsRegions } from 'helpers/contributions';
+
+export const annualAmountsControl: AmountsRegions = {
+  // Use the config from server
+};
+
+export const annualAmountsA: AmountsRegions = {
```

Comment:
if the number of actually overridden amounts is less unwieldy, I'd be up for putting this straight in `abtestDefinitions.js` so it's all in one place (just a single file to change when removing).

pending getting the actual amounts!